packages/datatrak-web/src/features/Survey/SurveyContext/utils.ts

```diff
@@ -155,6 +152,16 @@ const resetInvisibleQuestions = (
   return updatedFormData;
 };
 
+const hasConditionConfig = (
+  ssc: SurveyScreenComponent,
+): ssc is SurveyScreenComponent & { config: { condition: ConditionQuestionConfig } } =>
+  ssc.type === QuestionType.Condition && ssc.config?.condition !== undefined;
```

Comment:
@tcaiger Decided to include these type guards in the end. Happy with either approach in this context really, but in general prefer to avoid `as <Type>` :+1:

Reply:
Thanks @rohan-bes I totally agree and like this approach. @acdunham this is an example of the typeguards I mentioned the other day